client/src/components/derp.log

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Is this file needed? Or can we add it to .gitignore?

Reply:
Will remove